src/odb/test/write_lef_polygon.lefok

```diff
@@ -0,0 +1,2921 @@
+      LAYER metal1 ;
+        RECT  0.1375 0.2775 0.2025 0.9575 ;
+        RECT  0.1375 0.8925 0.6625 0.9575 ;
+        RECT  0.9975 1.1575 1.1325 1.2225 ;
+        RECT  0.5975 0.8925 0.6625 2.0875 ;
+        RECT  0.9975 1.1575 1.0625 2.0875 ;
+        RECT  0.1925 2.0225 1.0625 2.0875 ;
+        RECT  0.1925 1.2625 0.2575 2.2375 ;
+        RECT  1.4275 0.13 1.5825 0.195 ;
+        RECT  0.8675 1.0275 1.4925 1.0925 ;
+        RECT  0.8675 1.0275 0.9325 1.1725 ;
+        RECT  1.4275 0.13 1.4925 2.2375 ;
+    END
+END XNOR2X1
+
+MACRO XOR2X1
+    CLASS CORE ;
+    SIZE 1.71 BY 2.47 ;
+    SYMMETRY X Y ;
+    SITE CoreSite ;
+    PIN A
+        DIRECTION INPUT ; 
+        USE SIGNAL ; 
+        PORT
+            LAYER metal1 ;
+              RECT  0.5975 0.1675 0.6625 1.0125 ;
+              RECT  0.5975 0.1675 0.9325 0.2325 ;
+              RECT  0.8675 0.1675 0.9325 1.2225 ;
+              RECT  0.8675 1.1575 1.0025 1.2225 ;
+              RECT  0.8675 1.0275 1.3475 1.0925 ;
+              RECT  1.2825 1.0275 1.3475 1.3625 ;
+              RECT  1.2825 1.1575 1.4175 1.3625 ;
+        END
+    END A
+    PIN B
+        DIRECTION INPUT ; 
+        USE SIGNAL ; 
+        PORT
+            LAYER metal1 ;
+              RECT  0.2575 0.9175 0.4125 1.1125 ;
+        END
+    END B
+    PIN Y
+        DIRECTION OUTPUT ; 
+        USE SIGNAL ; 
+        PORT
+            LAYER metal1 ;
+              RECT  0.7275 0.2975 0.7925 1.5625 ;
+              RECT  0.7275 1.2875 0.8475 1.5625 ;
+        END
+    END Y
+    PIN gnd
+        DIRECTION INOUT ; 
+        USE GROUND ; 
+        PORT
+            LAYER metal1 ;
+              RECT  0.2925 -0.065 0.3575 0.8525 ;
+              RECT  1.1025 -0.065 1.1675 0.7125 ;
+              RECT  0 -0.065 1.71 0.065 ;
+        END
+    END gnd
+    PIN vdd
+        DIRECTION INOUT ; 
+        USE POWER ; 
+        PORT
+            LAYER metal1 ;
+              RECT  0.2375 1.7675 0.3025 2.535 ;
+              RECT  1.1975 1.4875 1.2625 2.535 ;
+              RECT  0 2.405 1.71 2.535 ;
+        END
+    END vdd
+    OBS
+      LAYER metal1 ;
+        RECT  0.0525 0.13 0.1875 0.195 ;
+        RECT  1.0675 1.1575 1.2025 1.2225 ;
+        RECT  1.0675 1.1575 1.1325 1.6925 ;
+        RECT  0.0525 1.6275 1.1325 1.6925 ;
+        RECT  0.0525 0.13 0.1175 2.2375 ;
+        RECT  1.5225 0.3375 1.6575 0.8225 ;
+        RECT  1.5225 0.3375 1.5875 0.9625 ;
+        RECT  1.1225 0.8975 1.6225 0.9625 ;
+        RECT  1.5575 0.8975 1.6225 2.2375 ;
+    END
+END XOR2X1
+
+MACRO BUMP
+    CLASS COVER BUMP ;
+    ORIGIN 14.5 14.5 ;
+    SIZE 29 BY 29 ;
+    PIN PAD
+        DIRECTION INOUT ; 
+        USE SIGNAL ; 
+        PORT
+            LAYER metal6 ;
+              POLYGON  14.5 -5.42 5.42 -14.5 -5.42 -14.5 -14.5 -5.42 -14.5 5.42 -5.42 14.5 5.42 14.5 14.5 5.42 ;
+        END
+    END PAD
+    OBS
+      LAYER via5 ;
+        POLYGON  14.5 -5.42 5.42 -14.5 -5.42 -14.5 -14.5 -5.42 -14.5 5.42 -5.42 14.5 5.42 14.5 14.5 5.42 ;
```

Comment:
The input LEF has
```
OBS
    LAYER via5 ;
      POLYGON 14.5 5.42 5.42 14.5 -5.42 14.5 -14.5 5.42 -14.5 -5.42 -5.42 -14.5 5.42 -14.5 14.5 -5.42 ;
  END
```
which doesn't match this polygon.

Reply:
In the decomp come we reorder the points to be "clockwise", I think it gets applied to the polygon, but shouldn't (I don't know if its needed any more)

Reply:
thanks for catching that, I moved the clockwise thing into the decompose code to avoid modifying the polygon points